### ISSUE TITLE
Unable to build with pgsql and axps2

### ISSUE THREAD

**Issue by cmizzi** [phpbrew/phpbrew#564]
Hi everyone. I look around all issues that could solve my problem but no solution was said, so I create a new issue to get help... Before starting, I'm running on OS X 10.10.

I'm trying to install PHP apache2 and postgresql support at the same time. Once the building is on the creation of the PHP apache module, I've got this error : 

```bash
Undefined symbols for architecture x86_64:
  "_lo_lseek64", referenced from:
      _zif_pg_lo_seek in pgsql.o
  "_lo_tell64", referenced from:
      _zif_pg_lo_tell in pgsql.o
  "_lo_truncate64", referenced from:
      _zif_pg_lo_truncate in pgsql.o
ld: symbol(s) not found for architecture x86_64
clang: error: linker command failed with exit code 1 (use -v to see invocation)
make: *** [libs/libphp5.6.12.bundle] Error 1
```

The problem is made by PostgreSQL (thought because of all read related posts). I installed PostgreSQL with Homebrew, by [this tap](https://github.com/petere/homebrew-postgresql). With it, I also install the PostgreSQL common library with symlinks creation.

I've tried to delete the PG-Common package and reset links to just use the basic package. It doesn't solve my issue. I also set the full path of my pgsql installation but there's nothing to do.

This is latest tests commands with my directory struct for postgres : 

```bash
phpbrew install 5.6.12 +default +openssl +mb +debug +gd +phar +apxs2 +mcrypt +pdo +pgsql=/usr/local/Cellar/postgresql-9.3/9.3.9/bin
phpbrew install 5.6.12 +default +openssl +mb +debug +gd +phar +apxs2 +mcrypt +pdo +pgsql=/usr/local/Cellar/postgresql-9.3/9.3.9
phpbrew install 5.6.12 +default +openssl +mb +debug +gd +phar +apxs2 +mcrypt +dbs
```

```bash
postgresql-9.3 [master] % find /usr/local/Cellar/postgresql-9.3 -type d -depth 2
/usr/local/Cellar/postgresql-9.3/9.3.9/bin
/usr/local/Cellar/postgresql-9.3/9.3.9/include
/usr/local/Cellar/postgresql-9.3/9.3.9/lib
/usr/local/Cellar/postgresql-9.3/9.3.9/share
```

What can I do ? Thank you for your help

**Comment by c9s** [comment]
try --debug flag, see which header & lib was used for building your php.  and you can use otool to see which lib was linked with the binary.

[Sent from iPhone]

**Comment by cmizzi** [comment]
I tried with an other configurations. I added the debug flag and this is it log : 

```bash
postgresql-9.3 [master] % phpbrew --debug install 5.6.12 +apxs2 +pgsql +pdo -- --with-pgsql=/usr/local/Cellar/postgresql-9.3/9.3.9/
===> phpbrew will now build 5.6.12
---> Parsing variants from command arguments '+apxs2 +pgsql +pdo -- --with-pgsql=/usr/local/Cellar/postgresql-9.3/9.3.9/'
===> Loading and resolving variants...
Checking distribution checksum...
Checksum matched: 4578dee9d979114610a444bee263ed9b
===> Distribution file was successfully extracted, skipping...
Source Directory: /Users/cyril/.phpbrew/build/php-5.6.12
Found existing Makefile, running make clean to ensure everything will be rebuilt.
You can append --no-clean option after the install command if you don't want to rebuild.
===> Running make clean: make -C /Users/cyril/.phpbrew/build/php-5.6.12 --quiet clean
Running Command:make -C /Users/cyril/.phpbrew/build/php-5.6.12 --quiet clean
Writing variant info to /Users/cyril/.phpbrew/php/php-5.6.12/phpbrew.variants
Enabled variants: apxs2, pgsql, pdo, xml
Disabled variants:
Found apxs2 binary: /usr/sbin/apxs
===> Applying patch - apxs2 module version name ...
Found existing build.log, renaming it to /Users/cyril/.phpbrew/build/php-5.6.12/build.log.1440266833
===> Configuring 5.6.12...


Use tail command to see what's going on:
   $ tail -F /Users/cyril/.phpbrew/build/php-5.6.12/build.log


./configure '--prefix=/Users/cyril/.phpbrew/php/php-5.6.12' '--with-config-file-path=/Users/cyril/.phpbrew/php/php-5.6.12/etc' '--with-config-file-scan-dir=/Users/cyril/.phpbrew/php/php-5.6.12/var/db' '--with-pear=/Users/cyril/.phpbrew/php/php-5.6.12/lib/php' '--disable-all' '--enable-phar' '--enable-session' '--enable-short-tags' '--enable-tokenizer' '--with-pcre-regex' '--with-zlib=/usr' '--with-apxs2=/usr/sbin/apxs' '--with-pgsql=../Cellar/postgresql-common/HEAD/bin/psql' '--with-pdo-pgsql=../Cellar/postgresql-common/HEAD/bin/psql' '--enable-pdo' '--enable-dom' '--enable-libxml' '--enable-simplexml' '--enable-xml' '--enable-xmlreader' '--enable-xmlwriter' '--with-xsl' '--with-libxml-dir=/usr' '--with-pgsql=/usr/local/Cellar/postgresql-9.3/9.3.9/' '--with-pgsql=/usr/local/Cellar/postgresql-9.3/9.3.9/' >> /Users/cyril/.phpbrew/build/php-5.6.12/build.log 2>&1
===> Building...
make >> /Users/cyril/.phpbrew/build/php-5.6.12/build.log 2>&1

PhpBrew\Exception\SystemCommandException: Make failed.
Thrown from phar:///usr/bin/phpbrew/PhpBrew/Tasks/BuildTask.php at line 2:

    1<?php
>   2namespace PhpBrew\Tasks; use PhpBrew\Exception\SystemCommandException; use PhpBrew\CommandBuilder; use PhpBrew\Build; class BuildTask extends BaseTask { public function run(Build $build, $targets = array()) { if ($build->getState() >= Build::STATE_BUILD) { $this->info("===> Already built, skipping..."); return; } $this->info("===> Building..."); $cmd = new CommandBuilder('make'); $cmd->setAppendLog(true); $cmd->setLogPath($build->getBuildLogPath()); $cmd->setStdout($this->options->{'stdout'}); if (!empty($targets)) { foreach ($targets as $t) { $cmd->addArg($t); } } if ($this->options->nice) { $cmd->nice($this->options->nice); } if ($makeJobs = $this->options->{'jobs'}) { $cmd->addArg("-j{$makeJobs}"); } $this->debug($cmd->getCommand()); if (!$this->options->dryrun) { $startTime = microtime(true); $code = $cmd->execute(); if ($code != 0) { throw new SystemCommandException('Make failed.', $build->getBuildLogPath()); } $buildTime = round((microtime(true) - $startTime) / 60, 1); $this->info("Build finished: $buildTime minutes."); } $build->setState(Build::STATE_BUILD); } }
    3

Trace:

    0) PhpBrew\Tasks\BuildTask->run(PhpBrew\Build)
        from phar:///usr/bin/phpbrew/PhpBrew/Command/InstallCommand.php: 2

    1) PhpBrew\Command\InstallCommand->execute('5.6.12', '+apxs2', '+pgsql', '+pdo', '--', '--with-pgsql=/usr/local/Cellar/postgresql-9.3/9.3.9/')
        from : 0

    2) call_user_func_array([PhpBrew\Command\InstallCommand, 'execute'], ['5.6.12', '+apxs2', '+pgsql', '+pdo', '--', '--with-pgsql=/usr/local/Cellar/postgresql-9.3/9.3.9/'])
        from phar:///usr/bin/phpbrew/CLIFramework/CommandBase.php: 2

    3) CLIFramework\CommandBase->executeWrapper(['5.6.12', '+apxs2', '+pgsql', '+pdo', '--', '--with-pgsql=/usr/local/Cellar/postgresql-9.3/9.3.9/'])
        from phar:///usr/bin/phpbrew/CLIFramework/Application.php: 2

    4) CLIFramework\Application->run(['/usr/bin/phpbrew', '--debug', 'install', '5.6.12', '+apxs2', '+pgsql', '+pdo', '--', '--with-pgsql=/usr/local/Cellar/postgresql-9.3/9.3.9/'])
        from phar:///usr/bin/phpbrew/CLIFramework/Application.php: 2

    5) CLIFramework\Application->runWithTry(['/usr/bin/phpbrew', '--debug', 'install', '5.6.12', '+apxs2', '+pgsql', '+pdo', '--', '--with-pgsql=/usr/local/Cellar/postgresql-9.3/9.3.9/'])
        from phar:///usr/bin/phpbrew/scripts/phpbrew-emb.php: 2

    6) require('phar:///usr/bin/phpbrew/scripts/phpbrew-emb.php')
        from /usr/bin/phpbrew: 6
```

I've also tried with the `bin` directory : same error.

**Comment by c9s** [comment]
I might need your `/Users/cyril/.phpbrew/build/php-5.6.12/build.log` content to debug the build process.
